src/module/server-instance/MemoryChart.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference path="../../../typings/globals/d3/index.d.ts" />
+/// <reference path="../../../typings/globals/c3/index.d.ts" />
+
+import * as c3 from "c3";
+export class InstanceMemoryChart {
+    chart:any;
+    constructor(used: number, max: number) {
+        this.chart = c3.generate({
+            bindto: '#chart',
+            size: {
+                width: 220,
+                height: 220
+            },
+            data: {
+                columns: [['Used', used], ['Free', max]],
+                type: 'donut'
+            },
+            donut: {
+                title:'Memory'
```

Comment:
Minor nitpick, but lets try to use double quotes in Typescript files, as per the rules of TSlint.  You can set webstorm to make these checks for you. Also, we should be able to call gulp check on the cmd line, however this isn't currently working, so I will look into that